components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -1654,6 +1655,68 @@ public OAuthRevocationResponseDTO revokeIssuedTokensByApplication(OAuthAppRevoca
         return revokeRespDTO;
     }
 
+    /**
+     * Revoke issued tokens for the application for the given authorized organization.
+     *
+     * @param application  {@link OAuthAppRevocationRequestDTO}.
+     * @param tenantDomain Tenant domain of the organization for which the tokens should be revoked.
+     * @return revokeRespDTO {@link OAuthAppRevocationRequestDTO}.
+     * @throws IdentityOAuthAdminException Error while revoking the issued tokens.
+     */
+    public OAuthRevocationResponseDTO revokeIssuedTokensForApplicationByOrganization
```

Comment:
better if we can pass the organization ID as the method suggests. Also we may not need to convert the tenant domain to org in later phases.
Also a suggestion for the method signature. `revokeIssuedTokensForOrganizationByApplication`